### DISCUSSION TITLE
WinGet and older Windows Versions

### DISCUSSION THREAD

**Discussion by Stockbruch** [amidaware/tacticalrmm#556] [General]
As i was reading through the issues, i saw that you want to switch from chocolatey to winget.
But as far as i could gather, winget seems to only work with Windows 10.
Does that mean, that in the future we will not be able to do software installations and updates on Windows 7 and 8?
I know that Windows 7 is not really supported anymore, but we still have some clients, that weren't able to upgrade.

**Comment by dinger1986** [comment]
Quite possibly.

This is more to do with winget than tactical and being fair winget is much better solution for software and updates than chocolatey really

**Reply by Stockbruch** [reply]
Alright, not a dealbreaker but something I have to keep in mind.
Would you know how winget would get implemented into tactical?
As we have many school clients it would be great if we were able to create repositories for each school
and have the possibility to install apps through the public and the local repositories.
But I know that that’s not as straightforward to implement as the solution tactical uses with chocolatey right now.

**Comment by bbrendon** [comment]
You can still use choco even if tactical doesn't support it directly.

**Comment by wh1te909** [comment]
i haven't really looked at winget yet but from what i saw in discord it doesn't appear to work from the system account in which case that's useless
